README.md

```diff
@@ -35,6 +33,11 @@ You can find an example of using this integration on [our website](https://dev.f
 
 You can find more info on the FingerprintJS Pro agent documentation page [https://dev.fingerprint.com/docs/js-agent](https://dev.fingerprint.com/docs/js-agent)
 
+`Tag type` - The way you want to use the tag. It is 3 options:
+  - `Load and identify` – default behavior. Load JS Agent and instantly identify browser.
+  - `Load` – just load JS Agent.
+  - `Identify` - identify browser. JS Agent needs to be loaded before. You can collect necessary data, then trigger this tag and provide `linkedId` or `tag`.
+
```

Comment:
Github doesn't allow you to comment on any line in the file just the changed ones? Weird. I wanted to comment on line 96, it seems `scriptUrlPattern` is supported now, should we remove it from Limitations?